templates/policy-compliance-agent/backend/services/vector_db_manager.py

```
@@ -0,0 +1,389 @@
+            if not chunks:
+                raise ValueError("No chunks provided to save")
+
+            # Generate unique document ID
+            document_id = str(uuid.uuid4())
+
+            # Prepare data for ChromaDB
+            ids = []
+            documents = []
+            metadatas = []
+
+            for i, (chunk) in enumerate(zip(chunks, embeddings)):
```

Ruff (0.14.5)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)